docs/source/json-api/index.rst

```diff
@@ -892,27 +924,41 @@ Some notes on behavior:
    results if you walk the array forwards, backwards, or in random
    order.
 
-Fetch All Known Parties
-=======================
+Fetch by Key Contracts Stream
+-----------------------------
 
-- URL: ``/parties``
-- Method: ``GET``
-- Content: <EMPTY>
+- URL: ``/stream/fetch``
+- Scheme: ``ws``
+- Protocol: ``WebSocket``
 
-HTTP Response
--------------
+List currently active contracts that match one of the given ``templateId - key`` pairs, with continuous updates.
 
-- Content-Type: ``application/json``
-- Content:
+Similarly to `Contracts Query Stream`_, two subprotocols must be passed, as described in `Choosing a party
+<#choosing-a-party>`__.
+
+``application/json`` body must be sent first, formatted according to the following rule:
+
+.. code-block:: none
+
+    [
+        {"templateId": "<template ID 1>", "key": <key 1>}
+        , {"templateId": "<template ID 2>", "key": <key 2>}
+        ...
+        , {"templateId": "<template ID N>", "key": <key N>}
+    ]
+
+Where:
+
+- ``templateId`` -- contract template identifier, same as in :ref:`create request <create-request>`,
+- ``key`` -- contract key, formatted according to the :doc:`lf-value-specification`,
+
+Example:
 
 .. code-block:: json
 
-    {
-        "status": 200,
-        "result": [
-            {
-                "party": "Alice",
-                "isLocal": true
-            }
-        ]
-    }
+    [
+        {"templateId": "Account:Account", "key": ["Alice", "abc123"]}
```

Comment:
```suggestion
        {"templateId": "Account:Account", "key": {"_1": "Alice", "_2": "abc123"}}
```
I assume you wanted a pair here. They are encoded as `{_1: ..., _2: ...}`. Getting a key that would actually produce an array would be quite weird in surface DAML.

Reply:
you can encode a pair both ways, I have a test that proves this.

Reply:
do you want the field names `_1` and `_2`? I will have to update the entire doc, because non streaming version of fetch and exercise by key, use an array to encode `Tuple2`

See LF JSON spec: https://docs.daml.com/json-api/lf-value-specification.html#record
> `[42, true]`
> The motivation for the array format for records is to allow specifying tuple types closer to what it looks like in DAML.

Reply:
You're absolutely right. However, since the JSON API always returns pairs in the format I described, it would be better to use that format in the docs and leave the other format as a convenience feature for our users.

Reply:
If that's the same everywhere in the docs, then let's be consistent for now and fix it in the whole doc in one go.

Reply:
#4418 created to address the above documentation issue. Keeping the docs consistent for now.